components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/client/authentication/OAuthClientAuthnService.java

```diff
@@ -60,6 +75,28 @@ public List<OAuthClientAuthenticator> getClientAuthenticators() {
     public OAuthClientAuthnContext authenticateClient(HttpServletRequest request, Map<String, List> bodyContentParams) {
 
         OAuthClientAuthnContext oAuthClientAuthnContext = new OAuthClientAuthnContext();
+        try {
+            String clientId = extractClientId(request, bodyContentParams);
+            if (!StringUtils.isBlank(clientId) && Boolean.parseBoolean(isFapiApplication(clientId))) {
```

Comment:
StringUtils.isNotBlank

Reply:
Fixed in 36cde69f01a0865dc525ff8c963f66e0af53cf89